ci/envs/310-oldest.yaml

```diff
@@ -2,17 +2,17 @@ name: test
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
-  - geopandas=0.8
+  - python=3.10
+  - geopandas=0.12
   - inequality
   - libpysal=4.6.0
   - mapclassify
-  - networkx=2.3
-  - numpy=1.21
+  - networkx=2.7
+  - numpy=1.22
   - packaging
-  - pandas>=0.23.0,!=1.5.0,<2
+  - pandas>=1.4.0,!=1.5.0,<2
   - shapely=2.0
-  - tqdm=4.27.0
+  - tqdm=4.63.0
```

Comment:
Is there some new functionality within `tqdm` that needs the update or is this simply following `SPEC0`?

Reply:
SPEC0